openstef_dbc/data_interface.py

```diff
@@ -138,9 +139,7 @@ def exec_influx_query(self, query: str, bind_params: dict = {}) -> dict:
             defaultdict: Query result.
         """
         try:
-            return self.influx_client.query(
-                query, bind_params=bind_params, chunked=True, chunk_size=10000
-            )
+            return self.influx_query_api.query_data_frame(query)
```

Comment:
1. No bind_params are used? I would expect that stuff using this function expect something to happen with the bind_params. Or if they are used nowhere, I guess they could be removed here as well. UPDATE: please use bind_params again. It is supported by the new client and considered a safer practise. (see comment further down for the link)

2. Are 'chunked' and 'chun_size' automagically fixed within the .query_data_frame?

Reply:
I fully agree with you comment, however the v1.8 does not support bind_params yet. They are only supported v>2, we will add them when v2 is live and release a new version of openstef_dbc with bind params.